source/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required(VERSION 3.13)
+
+set(CMAKE_C_STANDARD 99)
+
+# Add coreMQTT Lib
+add_library(coremqtt
```

Comment:
>coremqtt [](http://example.com/codeflow?start=12&length=8)

coreMQTT target has dependency on config folder, but we are not providing it ... Can you check if this build when only lib is build?